frontend/my-app/src/Pages/Account.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react'
+import './Account.css'
+import { useNavigate } from 'react-router-dom'
+
+const Account: React.FC = () => {
+  const navigate = useNavigate()
+  const [email, setEmail] = useState('')
+  const [location, setLocation] = useState('')
+  const [password, setPassword] = useState('')
+  const [passwordConfirm, setPasswordConfirm] = useState('')
+
+  const handleDeleteAccount = (): void => {
+    console.log('Delete Account dummy function')
+  }
+
+  const handleUploadImage = (): void => {
+    console.log('Upload Image dummy function')
+  }
+
+  const handleBack = (): void => {
+    console.log('Back dummy function')
+    navigate('/landing')
+  }
+
+  const handleUpdateEmail = (): void => {
+    console.log('Update Email dummy function')
+  }
+
+  const handleUpdateLocation = (): void => {
+    console.log('Update Location dummy function')
+  }
+
+  const handleUpdatePassword = (): void => {
+    console.log('Update Password dummy function')
+  }
+
+  return (
+        <div className="container">
+        <div className="left-column">
+            <button className="upload-image" onClick={handleUploadImage}>
+                Upload Image
+            </button>
+            <button className="delete-account" onClick={handleDeleteAccount}>
+                Delete Account
+            </button>
+            <button className="back" onClick={handleBack}>
+                Back
+            </button>
+        </div>
+        <div className="right-column">
```

Comment:
This feels like a lot of things that we could (and should) break down into multiple components, but for the purposes of getting a beta version out it looks good